content/national-qualifications-framework/ncit/agile-meetings/_index.md

```diff
@@ -1,6 +1,38 @@
 ---
 _db_id: 357
 content_type: topic
+flavours:
```

Comment:
rahter dont add flavours unless the project actually has flavours.

Is this coded? Does the language matter? 
Does this have a prerequisite that is coded? 

If you add flavours here then flavours need to be specified elsewhere and it has a knock on efect. It's better to be accurate and specific. No spray and pray